tokio-postgres/src/client.rs

```diff
@@ -81,6 +84,21 @@ impl InnerClient {
         })
     }
 
+    // Send a message for the existing entry in the pipeline; don't
+    // create a new entry in the pipeline. This is needed for CopyBoth
+    // mode (i.e. streaming replication), where the client may send a
+    // new message that is part of the existing request.
+    pub fn unpipelined_send(&self, messages: RequestMessages) -> Result<(), Error> {
```

Comment:
Why would CopyBoth need a different API to do this than the existing CopyIn logic?